java/README.md

```diff
@@ -90,20 +58,130 @@ Other useful gradle developer commands:
 * `./gradlew :examples:run` to run client examples (make sure you have a running redis on port `6379`)
 * `./gradlew :benchmarks:run` to run performance benchmarks
 
-## Basic Examples
 
-### Standalone Redis:
+### Setting up the Driver
+
+Refer to https://central.sonatype.com/search?q=glide&namespace=software.amazon.glide for your specific system.
+Once set up, you can run the basic examples.
 
+Gradle:
+- Copy the snippet and paste it in the `build.gradle` dependencies section.
+Example shown below is for `glide-osx-aarch_64`.
```

Comment:
This statement references `glide-osx-aarch_64` but the code snippet below uses `glide-for-redis` as the package name. This does not seem to match what I find when I follow the Maven repo link above. There are packages with architecture-specific names, including a `software.amazon.glide/glide-osx-aarch_64@0.4.2` package. If this example is correct, does it need some additional explanation to avoid confusion?